build.gradle

```diff
@@ -10,3 +12,13 @@ dependencies {
     compile localGroovy()
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withJavadocJar()
+    withSourcesJar()
+}
+
+repositories {
```

Comment:
Do we need this repo adding? Doesn't look like it has any dependencies so shouldn't need to pull anything? Or is it for the plugion?

Reply:
I was following the documentation here (https://verify-team-manual.cloudapps.digital/support/third-line/library-publishing/#working-example). It sounds like its needed to be able to control and whitelist sources for the project

Reply:
I think this was included in those instructions as the project they were based on has dependencies that need pulling in from the repos defined in this block. This project only has deps that don't need a remote repo defined (hence why it works on master without this block), so you don't need to add this. I've just tried whipping it out locally and it seems fine.